client/boot/index.js

```diff
@@ -165,14 +166,7 @@ function boot() {
 function renderLayout( reduxStore ) {
 	const Layout = require( 'controller' ).ReduxWrappedLayout;
 
-	const layoutElement = React.createElement( Layout, {
-		store: reduxStore
-	} );
-
-	ReactDom.render(
-		layoutElement,
-		document.getElementById( 'wpcom' )
-	);
+	renderWithReduxStore( React.createElement( Layout, { store: reduxStore } ), 'wpcom', reduxStore );
```

Comment:
Yes, it looks like it is wrapped twice: https://github.com/Automattic/wp-calypso/blob/master/client/controller/index.node.js#L19.